src/components/molecules/TheHeader.vue

```diff
@@ -1,33 +1,120 @@
+            Français
+            <!-- {language === "en" ? "English" : "Français"} -->
+          </a>
 
-          <!-- Brand Logo -->
-          <img class="h-10" alt="Vue logo" src="../../assets/logo.png">
+          <!-- end lang toggle desktop -->
+        </div>
 
-          <!-- Site title -->
-           <div>
-             <p class="text-gray-800 font-semibold">Site Title</p>
-           </div>
+        <!-- start gc logo -->
+        <!-- src={language === "en" ? "../../assets/sig-blk-en.svg" : "../../assets/sig-blk-fr.svg"} -->
+        <div
+          className="flex flex-col sm:flex-row container w-full mx-auto p-1 justify-between items-center"
+        >
+          <div className="flex flex-row items-center w-full justify-between">
+            <img
+              className="h-7  sm:h-8 md:h-8 mx-3 sm:mx-0 "
+              src="../../assets/sig-blk-en.svg"
+              alt="canada logo en"
+            />
+            <a id="lang-toggle-small"
+              class="
+                sm:hidden
+                m-3
+                sm:m-0
+                font-semibold font-body
+                underline
+              "
+            >
+              FR
+              <!-- {language === "en" ? "EN" : "FR"} -->
+            </a>
+          </div>
+          <!-- end gc logo -->
+          <!-- start gc search bar -->
+          <div className="inline-flex w-full sm:w-80 sm:h-10">
+            <form
+              className="w-full inline-flex border border-solid border-gray-light-100 mx-2 sm:mx-0"
+            >
+              <input
+              id="searchbar"
+               aria-label="Search"
+                type="text"
+                placeholder="Search Canada.ca"
+                className="w-full placeholder-text-gray text-text-gray font-body py-1 px-2 focus:outline-none"
+              />
 
+              <button
+              id="searchbutton"
+                title="Search bar button"
+                type="submit"
+                class="
+                  bg-primary-main
+                  text-white text-center
+                  p-2
+                  rounded-none
+                  hover:bg-gray-default
+                  active:bg-gray-default
+                  focus:bg-gray-default
+                "
+              >
+                <span id="search-icon">
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    class="h-6 w-6"
+                    fill="none"
+                    viewBox="0 0 24 24"
+                    stroke="currentColor"
+                  >
+                    <path
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                      stroke-width="2"
+                      d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
+                    />
+                  </svg>
+                </span>
+              </button>
+            </form>
+          </div>
```

Comment:
The search bar can be hidden in mobile

Reply:
I asked Thomas about the hidden or not hidden on mobile and I mentioned that I had it already showing on mobile (it was one of my questions) and he said to leave it showing on mobile.

Reply:
Sounds good!